src/Main/Main.cc

```diff
@@ -74,9 +74,8 @@ int main(int argc, char* argv[]) {
             default:
                 carta_log_sink = new CartaLogSink(casacore::LogMessage::NORMAL);
         }
-        casacore::LogSink log_sink(carta_log_sink->filter(), std::shared_ptr<casacore::LogSinkInterface>(carta_log_sink));
         casacore::LogSink::globalSink(carta_log_sink);
-        casacore::LogIO casacore_log(log_sink);
+        casacore::LogIO casacore_log;
```

Comment:
LogIO not needed?

Reply:
Good point; this object is actually never used.